.travis.yml

```diff
@@ -34,7 +34,8 @@ matrix:
         - npm install -C dex-contracts
         # driver/listener require compiled contracts
         - cd dex-contracts && truffle compile && cd -
-        - docker-compose up -d driver listener | while read line ; do echo "$(date)| $line"; done; # prints timestamps for profiling
+        - docker-compose up -d ganache-cli driver listener | while read line ; do echo "$(date)| $line"; done; # prints timestamps for profiling
```

Comment:
Hasn't this change already been pushed to Master?

Reply:
Yeah...
also I only made this commit to get travis started again, but travis is a bitch today